src/ActionBar/_ActionBarActions.js

```diff
@@ -0,0 +1,19 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ActionBarActions = ({ children, className, ...props }) => {
+    const actionBarActionsClasses = classnames(
+        'fd-action-bar__actions',
+        className
+    );
+
+    return <div {...props} className={actionBarActionsClasses}>{children}</div>;
+};
+
```

Comment:
Should non-exported components have a `displayName` set?  Just gathering opinions as I'm not quite sure myself if they should or not.

Reply:
Update:  I believe they should and I'll explain why.  In order to scan the component for sub-components, there needs to be some sort of property we can check against to validate this _is_ a component.  I think `displayName` should be that property.  I originally was thinking `propTypes`, but there's no guarantee every component has `propTypes` (especially sub-components).